src/components/Restoration/SummaryRestoreBusiness.vue

```diff
@@ -51,14 +51,17 @@
           </v-col>
 
           <v-col cols="12" sm="9">
-            <template v-if="isLimitedRestoration">
+            <template v-if="isLimitedRestorationFiling">
               <p id="limited-restoration">Limited Restoration</p>
               <p id="expires-on">Expires on {{ expiry }}</p>
             </template>
-            <template v-else>
+            <template v-else-if="isFullRestorationFiling">
               <p id="full-restoration">Full Restoration</p>
               <p id="applicants-relationships">Applicant's relationship(s): {{ relationships }}</p>
             </template>
+            <template v-else>
+              <p id="unknown-restoration-type">[Unknown]</p>
+            </template>
```

Comment:
This _shouldn't_ happen because we always initialize the restoration type, but just in case, it's incorrect to say "if it's not limited then it must be full", so now it's handled.

Reply:
I'll be fixing that actually. The default state is going to change to nothing being selected. So, this is great.